nasademo/src/main/resources/db/migration/V2_addPicturesTable.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS pictures
+(
+    id         BIGINT,
+    nasa_id    BIGINT,
```

Comment:
not null restrictions on all colums except id and nasa_id should have unique constraint